src/devices/bus/a2bus/uthernet.cpp

```diff
@@ -0,0 +1,77 @@
+// license:BSD-3-Clause
+// copyright-holders:R. Belmont
+/*********************************************************************
+
+    uthernet.cpp
+
+    Apple II Uthernet Card
+
+*********************************************************************/
+
+#include "emu.h"
+#include "emuopts.h"
+#include "uthernet.h"
+
+/***************************************************************************
+    PARAMETERS
+***************************************************************************/
+
+//**************************************************************************
+//  GLOBAL VARIABLES
+//**************************************************************************
+
+DEFINE_DEVICE_TYPE(A2BUS_UTHERNET, a2bus_uthernet_device, "uthernet", "Uthernet")
```

Comment:
Please add the `a2` prefix used by all the Apple II cards to the short name (third argument) – it should be `"a2uthernet"`.